articles/connecting-a-gitlab-respository-as-your-source.mdx

```diff
@@ -25,10 +25,33 @@ To connect a GitLab repository and start syncing files, follow these instruction
 
 <comp.DocsImage path="ye_olde_images/files-source-syncing-gitlab.png" alt="Selecting GitLab as your source provider" type="screenshot" />
 
-This redirects you to GitLab. Log in and authorize CloudCannon access to your GitLab account.
-You'll be redirected back to CloudCannon to pick a repository to connect. If you don't have one for this website, create a new one in GitLab and refresh this page.
+This will open a menu where you can connect to GitLab, via one of two methods.
 
-<comp.DocsImage path="ye_olde_images/files-source-syncing-gitlab-gitlab-selected.png" alt="Selecting your GitLab repository to sync" type="screenshot" />
+## Connecting with your user account
+
+You can connect to GitLab by authenticating with your user account. This will give your CloudCannon organization access to the same GitLab repositories you have access to.
+
+This option will redirect you to GitLab. Log in and authorize CloudCannon access to your GitLab account. You'll be redirected back to CloudCannon to pick a repository to connect.
+
+<comp.DocsImage path="ye_olde_images/files-source-syncing-gitlab-oauth.png" alt="A menu with a button labelled 'Authenticate with GitLab'." type="screenshot" />
+
+## Authenticating with a group access token
+
+Alternatively, you can create a group access token in GitLab and save that token in CloudCannon. This will give your CloudCannon organization access to all the resources scoped to your token.
```

Comment:
```suggestion
Alternatively, you can create a group access token in GitLab and save that token in CloudCannon. This will give your CloudCannon organization access to all the resources scoped to your token.

See GitLab's [Group access tokens](https://docs.gitlab.com/ee/user/group/settings/group_access_tokens.html) documentation for how to create and configure a token.

<comp.Notice info_type="important">
  For full functionality on CloudCannon, your group access token should have (at least) the Maintainer role, and the `api`, `read_api`, `read_repository`, and `write_repository` scopes.
</comp.Notice>
```